pkg/controller/controller.go

```diff
@@ -63,37 +68,11 @@ func NewController(shareRelist time.Duration) (*Controller, error) {
 	// informers.NewSharedInformerFactoryWithOptions, but we restrict OpenShift
 	// "system" namespaces with chatty configmaps like the leaderelection related ones
 	// that are updated every few seconds
-	//TODO make this list externally configurable
```

Comment:
```
tweakListOptions := internalinterfaces.TweakListOptionsFunc(func(options *metav1.ListOptions) {
        ignored := []string{}
	for _, ns := range ignoredNamespaces {
		ignored = append(ignored, fmt.Sprintf("metadata.namespace!=%s", ns))
	}
        options.FieldSelector = strings.Join(ignored, ",")
})
```

Reply:
This would be cleaner than all the appending strings stuff.

Reply:
true ... technically a tech debt item given @otaviof did not change that existing bit

I'm fine if he want to take that on now, or somebody does this as a follow up PR

Reply:
Thanks for the heads up, @coreydaley, it's more readable this way. However, I would argue this change should be tackled in a subsequent PR, would be "a good first issue" by the way.

Reply:
@coreydaley I'm bringing that up on #60, thanks for the suggestion!